db/good_morning.py

```diff
@@ -135,3 +135,25 @@ def reset_all_morning_points(session: sessionmaker):
         sess.execute(
             update(MorningPoints).values(weekly_count=0, timestamp=RESET_TIMESTAMP)
         )
+
+
+def manual_increment_morning_points(value: int, session: sessionmaker):
+    """Manually increment ALL USERS' morning points totals
+
+    Args:
+        value (int): Amount to increment users' morning points total by
+        session (sessionmaker): Open DB session
+    """
+    update_vals = []
+    with session() as sess:
+        all_morning_points: list[MorningPoints] = sess.query(MorningPoints).all()
```

Comment:
is this the best way to do this? not sure if sqlalchemy has a nicer way similar to SQL, i.e.

```sql
UPDATE table SET field = field + 1;
```

Reply:
I looked around for something like this and couldn't find it. This was basically the post that I modeled this solution after:
https://stackoverflow.com/questions/2334824/how-to-increase-a-counter-in-sqlalchemy

I have no idea if there's a way to get this to generate simple SQL to do this though. Lemme go dig a bit

Reply:
LOL good catch, somehow I missed this entire method on the `Query` object:
https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.update